src/main/java/com/iota/iri/network/protocol/Protocol.java

```diff
@@ -123,6 +128,15 @@ public static ByteBuffer createHeartbeatPacket(Heartbeat heartbeat) {
         return buf;
     }
 
+    public static ByteBuffer createMilestoneRequestPacket(MilestoneRequest milestoneRequest){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Public method and constructor comments are required](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=5322997)